third_party/move/move-compiler-v2/tests/more-v1/v2-not-supported/no_receiver_calls.move

```diff
@@ -0,0 +1,10 @@
+module 0x42::m {
+    struct T has drop {}
+
+    fun receiver(self: &T, _x: u64) { abort 1 }
+
+    fun call_receiver(t: T) {
+        t.receiver(1);
```

Comment:
Should these be run with language version 1?

Reply:
I put a copy in move-compiler-v2/tests/checking-lang-v1/.  That one looks more like the V1 test.